x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -67,36 +67,40 @@ export const getFieldItems = ({
   );
 };
 
-const getDefaultFieldTableColumns = (highlight: string): FieldTableColumns => [
-  {
+const getDefaultFieldTableColumns = ({
+  highlight,
+  shouldShowDescriptionColumn,
+}: {
+  highlight: string;
+  shouldShowDescriptionColumn: boolean;
```

Comment:
nitpick: maybe we can just call this `showDescriptionColumn`? Also, do you think we can have this as an optional parameter and default it to something? Let me know if that's a good idea or not haha